server/src/test/java/org/apache/iotdb/db/metadata/mtree/MTreeAboveSGTest.java

```diff
@@ -289,4 +292,49 @@ public void testGetNodeListInLevel() throws MetadataException {
     Assert.assertEquals(0, result.left.size());
     Assert.assertEquals(1, result.right.size());
   }
+
+  @Test
+  public void testSerialization() throws Exception {
+    PartialPath[] pathList =
+        new PartialPath[] {
+          new PartialPath("root.sg"),
+          new PartialPath("root.a.sg"),
+          new PartialPath("root.a.b.sg"),
+          new PartialPath("root.a.a.b.sg")
+        };
+    for (int i = 0; i < pathList.length; i++) {
+      root.setStorageGroup(pathList[i]);
+      IStorageGroupMNode storageGroupMNode =
+          root.getStorageGroupNodeByStorageGroupPath(pathList[i]);
+      storageGroupMNode.setDataTTL(i);
+      storageGroupMNode.setDataReplicationFactor(i);
+      storageGroupMNode.setSchemaReplicationFactor(i);
+      storageGroupMNode.setTimePartitionInterval(i);
+    }
+
+    ByteBuffer byteBuffer = ByteBuffer.allocate(1024 * 1024);
+    root.serialize(byteBuffer);
+    byteBuffer.flip();
+
+    root.clear();
+
+    root.deserialize(byteBuffer);
```

Comment:
How about new a MTreeAboveSG, then do deserialization for it? This is how the ConfigNode use it~

Reply:
Fixed.